src/main/resources/edu/arizona/sista/reach/biogrammar/events_master.yml

```diff
@@ -94,7 +94,7 @@ rules:
   # Generic (incomplete) events
   - import: edu/arizona/sista/reach/biogrammar/coref/generic_events.yml
     vars:
-      priority: "5"
```

Comment:
These no longer need to be quoted.  I should open an issue to replace all of these priorities with a variable.